### PR TITLE
Add support for youtube hash time index

### PR DESCRIPTION
**Pull request by mpatnode**
Support URLs like this:  https://www.youtube.com/watch?v=SJj_qy8hQ2E#t=243
Which jumps 243 seconds into the video

### CODE REVIEW

File: src/Pass/IframeYouTubeTagTransformPass.php [hunk 1]
```diff
@@ -114,7 +124,36 @@ protected function isYouTubeIframe(DOMQuery $el)
...
+     * Get the youtube anchor hash (if available)
+     *
+     * @param DOMQuery $el
+     * @return string
+     */
+    protected function getYouTubeCodeHash(DOMQuery $el) {
+        $youtube_code = $this->_getYouTubeCode($el);
+        // Return hash-tag timestamp jump:  #t=243
+        $youtube_code = explode('#', $youtube_code);
+        return $youtube_code[1] ?? NULL;
```
Comment (mpatnode) [inline src/Pass/IframeYouTubeTagTransformPass.php:146]: PHP 7 assumption here.   Is that bad?

Comment (SGudbrandsson) [reply src/Pass/IframeYouTubeTagTransformPass.php:146]: Yes ..
https://travis-ci.org/Lullabot/amp-library/builds/197507612

Comment (SGudbrandsson) [reply src/Pass/IframeYouTubeTagTransformPass.php:146]: We couldn't wait for the Lullabot team to wake up (I tried sending them an e-mail but they're busy..), so we forked them and started fixing bugs in another repo.
https://github.com/GuideToIceland/amp-library

You can use that to get better AMP conversion code.

**Comment by mpatnode** [comment]
A more complete solution is available in this branch:  https://github.com/GuideToIceland/amp-library

**Pull request closed without merging**